.github/labeler.yml

```diff
@@ -1,2 +1,100 @@
-Proposal:
+# Proposals
+proposal:
 - proposals/text/*
+
+# Topics
+topic:tests:
+- test/**/*
+- test/*
+- conftest.py
+topic:docker:
+- docker/*
+- docker-compose.yml
+topic:CI:
+- .github/*
+- .github/**/*
+topic:DX:
+- CONTRIBUTING.md
+- .pre-commit-config.yaml
+- .gitignore
+topic:dependencies:
+- pyproject.toml
+topic:build/distribution:
+- pyproject.toml
+topic:modeling:
+- haystack/modeling/*
+- haystack/modeling/**/*
+- test/modeling/*
+topic:reader:
+- haystack/nodes/reader/*
+- test/nodes/test_reader.py
+topic:retriever:
+- haystack/nodes/retriever/*
+- test/nodes/test_retriever.py
+topic:pipeline:
+- haystack/pipelines/*
+- test/pipelines/*
```

Comment:
`haystack/nodes/other/*` and `test/nodes/test_other.py` fit here too. They aren't mentioned anywhere else.